assets/stylesheets/bootstrap/_variables.scss

```diff
@@ -78,6 +78,9 @@ $font-size-h4:            ceil(($font-size-base * 1.538461538)) !default; // ~20
 $font-size-h5:            $font-size-base !default;
 $font-size-h6:            $font-size-base !default;
 
+$font-size-scale:         0.85;
```

Comment:
Although - as covered in Hipchat - this actually has no effect on our textboxes. The additions are only used to the heading scale, not body text.